### DISCUSSION TITLE
Break until next

### DISCUSSION THREAD

**Discussion by TiltedChris** [ThioJoe/Auto-Synced-Translated-Dubs#36] [General]
I got following error, how can I fix this
subsDict[str(int(line)-1)]['break_until_next'] = processedTime1 - int(subsDict[str(int(line) - 1)]['end_ms'])
                                                                          ~~~~~~~~^^^^^^^^^^^^^^^^^^^^
KeyError: '1'

**Comment by TharunDharmaraj** [comment]
> I got following error, how can I fix this
> subsDict[str(int(line)-1)]['break_until_next'] = processedTime1 - int(subsDict[str(int(line) - 1)]['end_ms'])
>                                                                           ~~~~~~~~^^^^^^^^^^^^^^^^^^^^
> KeyError: '1'

I would recommend you yo check your SRT file. I guess the timing would be mis alligned. It should follow the proper format.

**Comment by ThioJoe** [comment]
Try using the latest version of main.py:  https://github.com/ThioJoe/Auto-Synced-Translated-Dubs/blob/main/main.py
